### DISCUSSION TITLE
Contact Yahoo about bad price data - best way?

### DISCUSSION THREAD

**Discussion by ValueRaider** [ranaroussi/yfinance#1446] [General]
Has anyone successfully contacted Yahoo about bad price data, for them to fix it? I'm sure I've seen someone email before, or use a message form, but I can't find it.

**Comment by excorp11** [comment]
Bad / missing price data can be reported here:  https://yahoo.uservoice.com/forums/382977-finance?category_id=208525

**Comment by manuelwithbmw** [comment]
I have sometimes contacted <finance@cc.yahoo.com> about bad prices/bad data on equities and always got a response in 1 day or 2

**Reply by ValueRaider** [reply]
How long until they actually fix it? I emailed 2 weeks ago and got quick reply, but the data is still bad.

**Reply by manuelwithbmw** [reply]
I emailed them a dozen times in 2-3 years and always took them more or less 1 week to reply. Sometimes when they replied the data appeared to them to already be fixed, so I guess they have some sort of automatism or data refresh as well. Overall they never replied later than a few days, 2 weeks appear unusual to me. They always replied though

**Comment by manuelwithbmw** [comment]
Checked my last emails: I emailed them 17 Feb and replied 21st Feb. Then I emailed them again for another data issue on the 29 March and haven't replied yet.
